codespell_lib/data/dictionary.txt

```diff
@@ -51461,6 +51466,7 @@ smaple->sample
 smapled->sampled
 smaples->samples
 smapling->sampling
+sme->same, some,
```

Comment:
```suggestion
sme->same, seme, some,
```
However, SME is a valid abbreviation for a Small and Medium Enterprise. Perhaps move to the _rare_ dictionary?

Reply:
Personally I think that if SME is valid then it's something to add to the local ignore list.

Reply:
Updated the line, also added semes to somes .

Reply:
SME can be found in the [OED](https://www.oed.com/search/dictionary/?q=SME) or [Cambridge Dictionary]([url](https://dictionary.cambridge.org/dictionary/english/sme?q=SME)), it's quite common.

Reply:
I'll make the updates you wish - should I move it or not ?  IHMO, in code it will probably occur in a dictionnary, even just in a database initialisation and then be referred to as 'company_type' or similar.

Reply:
I've tried:
* https://grep.app/search?q=SME
* https://github.com/search?q=SME&type=repositories

I agree `SME`, is rare but it does occur, although it's usually the plural form (`SMEs`) or `SME` as an [ARM processor instruction](https://developer.arm.com/documentation/ka005091/1-0/?lang=en).

Reply:
Thanks for that search - apparently it occurs in just less than 13k repositories out of about 28M public repostories, so 0.5%.  And in those repositories it probably concerns only a few files.  I compared to "hass" which I've added to the ignore list in several projects - it occurs slightly more (19k repositories).

Anyway, do you want me to move it to the "rare" dictionnary or leave it in the base dictionnary?

Reply:
Let's leave it in the main dictionary for now, and wait for possible feedback.